pages/onboarding/index.tsx

```diff
@@ -131,6 +147,11 @@ const OnboardingPage: NextPage<Props> = ({ user }: Props) => {
           </Button>
         </Link>
       </Box>
+      <Snackbar open={openAlertError} autoHideDuration={6000} onClose={handleClose}>
```

Comment:
nit: I'd probably opt to have them in the bottom-right rather than the bottom-left. Personal opinion though...

Reply:
yeah idk i liked it in the left. i guess we can consult design team?

Reply:
Fair, I believe my bias may come from operating systems as they tend to default to top-right or bottom-right.

Anyway, this is non-blocking and can always be changed later easily enough. 👍🏼